.github/workflows/ci.yml

```diff
@@ -0,0 +1,50 @@
+name: CI
+
+on:
+  push:
+    branches: [ main, staging, trying ]
+  pull_request:
+    branches: [ main ]
+
+env:
+  CARGO_TERM_COLOR: always
+  TARGET: thumbv7em-none-eabihf
+  HAL: nrf52840-hal = "0.12.0"
+
+# NOTE if you change the `matrix` field or add/remove a job you'll need to update the list of jobs
+# in `.github/bors.toml
+jobs:
+  test:
+    strategy:
+      matrix:
+        rust:
+          - stable
+          - nightly
+    runs-on: ubuntu-latest
+    timeout-minutes: 20
+    steps:
+    - uses: actions/checkout@v2
+      with:
+        path: app-template
+    - uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: ${{ matrix.rust }}
+        override: true
+        target: ${{ env.TARGET }}
+    - name: debug
+      run: cd app-template && git status && ls -la
```

Comment:
here you `cd` into the `app-template`, I'm not sure if that would not affect further steps, so that below the `cargo generate --git app-template --name out` would fail because you are in the folder already?

Reply:
hmm, doesn't look like that's the cause either